package.json

```diff
@@ -621,6 +621,11 @@
           "group": "2_main@1",
           "submenu": "ansible.playbook.run",
           "when": "isFileSystemResource && editorLangId == ansible"
+        },
+        {
+          "group": "2_main@1",
+          "command": "ansible.lightspeed.playbookExplanation",
+          "when": "editorTextFocus && redhat.ansible.lightspeedExperimentalEnabled"
```

Comment:
Not that I know what I'm doing; but did you also want to include `&& resourceLangId == ansible`?

(Looking at the "Run Ansible Playbook via..." [definition](https://github.com/ansible/vscode-ansible/blob/945a907177a8be6d2c87dd20ba7bce3e8ab84a68/package.json#L620-L624)).